README.md

```diff
@@ -205,6 +205,7 @@ SparkSQL has [serveral built-in Data Sources](https://spark.apache.org/docs/late
 
 - [jupyter/docker-stacks/pyspark-notebook](https://github.com/jupyter/docker-stacks/tree/master/pyspark-notebook) - PySpark with Jupyter Notebook and Mesos client.
 - [sequenceiq/docker-spark](https://github.com/sequenceiq/docker-spark) - Yarn images from [SequenceIQ](http://www.sequenceiq.com/).
+- [datamechanics/spark](https://hub.docker.com/r/datamechanics/spark) - an easy to setup Docker image for Apache Spark from [Data Mechanics](https://www.datamechanics.co/).
```

Comment:
Nit. Let's change "an easy"  to "An easy" ‒ we capitalize leading articles for other entries so it will be better to avoid divergence here.

Reply:
I also agree with @zero323 on this.

Reply:
Thank you! Fixed in https://github.com/awesome-spark/awesome-spark/pull/219/commits/204c0accc91d27e99b9fb2b941e94e70187286f1